src/core_modules/capture-core/components/WidgetIndicator/WidgetIndicatorContent/WidgetIndicatorContent.js

```diff
@@ -86,7 +86,7 @@ const WidgetIndicatorContentComponent = ({ widgetData, emptyText, classes }: Con
             className={cx(classes.indicatorRow, { isLastItem })}
         >
             <div>{indicator.key}</div>
-            {(indicator.value || indicator.color) && (
+            {(String(indicator.value) || indicator.color) && ( // convert to string to make sure 0 gets displayed
```

Comment:
The flow type definition says that the indicator value should be a string when passed on to the Widget. I feel that makes sense and think it would be better to do this conversion in the rules engine itself. The resulting value from the following program rule actions should be converted to string in the rules engine: Display key/value pair, display text, error on complete, show error, show warning, warning on complete.

@Bekkalizer: Concerns?

Reply:
It seems sensible that items to display should always be converted to text in the engine.